formats/base64/commonMain/src/kotlinx.serialization.base64/ByteArrayAsBase64StringSerializer.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.serialization.base64
+
+import kotlinx.serialization.*
+import kotlinx.serialization.base64.impl.*
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.encoding.*
+
+/**
+ * Serializer that encodes and decodes [ByteArray] using [Base64](https://en.wikipedia.org/wiki/Base64) encodings.
+ * This is usually makes sense with text formats like JSON.
+ */
+public object ByteArrayAsBase64StringSerializer : KSerializer<ByteArray> {
+    override val descriptor: SerialDescriptor = PrimitiveSerialDescriptor(
```

Comment:
I come up with the same idea today before I see this PR. So this is a good feature. On the other hand, can we find a shorter name?
i.e:
`ByteArrayBase64Serializer`
or `Base64BytesSerializer`
or `Base64Serializer`
or `ByteArraySerializer` (replacing existing solution.)

Reply:
Well, there are already similar serializers in the library with same naming schema so I see no reason to break it

Reply:
the only example I can find was `LongAsStringSerializer` among 30+ other serializers. Since `Base64` already implies `String` we can at least reduce it to `ByteArrayAsBase64Serializer`. Anyway, it is up to you. It is your contribution(which I like), mine is just a mere suggestion.